google/cloud/bigquery/client.py

```diff
@@ -2647,9 +2668,19 @@ def load_table_from_dataframe(
                         job_config.schema,
                         tmppath,
                         parquet_compression=parquet_compression,
+                        parquet_use_compliant_nested_type=parquet_use_compliant_nested_type,
                     )
                 else:
-                    dataframe.to_parquet(tmppath, compression=parquet_compression)
+                    dataframe.to_parquet(
+                        tmppath,
+                        engine="pyarrow",
+                        compression=parquet_compression,
+                        **{
+                            "use_compliant_nested_type": parquet_use_compliant_nested_type
```

Comment:
Do we need to avoid passing this in if the pandas version doesn't support it? Or does pandas just pass through to pyarrow in older versions of pandas?

Reply:
I've opened a PR with them about this same thing but yes, currently it goes straight through to `pyarrow` so need to catch it.